aws/src/main/java/org/apache/iceberg/aws/s3/AnalyticsAcceleratorUtil.java

```diff
@@ -83,6 +95,16 @@ private static S3SeekableInputStreamFactory createNewFactory(
     return new S3SeekableInputStreamFactory(objectClient, streamConfiguration);
   }
 
+  private static void handleRemoval(S3SeekableInputStreamFactory factory) {
```

Comment:
this can probably just be inlined, otherwise the naming of the method is a bit misleading, because we're not handling removal inside the method but closing the factory. Alternatively just rename the method to `close(..)`

Reply:
Agree, I'll update it to `close` then.